_posts/2017-09-18-impl-future-for-rust.md

```diff
@@ -0,0 +1,312 @@
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-docs-rustdoc2</b></td>
+    <td>Get in on a bottom-up revamp of rustdoc!</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-docs-rbe</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+</table>
+
+### Dev tools team
+
+<table>
+<tr>
+    <td><b>WG-dev-tools-rls</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-dev-tools-vscode</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-dev-tools-rustfmt</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-dev-tools-IntelliJ</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-dev-tools-rustup</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-dev-tools-clippy</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-dev-tools-bindgen</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+</table>
+
+### Cargo team
+
+<table>
+<tr>
+    <td><b>WG-cargo-native</b></td>
+    <td>Let's make native dependencies as painless as we can.</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-cargo-registries</b></td>
+    <td>Going beyond crates.io to support custom registries.</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-cargo-pub-deps</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-cargo-integration</b></td>
+    <td>How easy can we make it to use Cargo with your build system?</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+</table>
+
+### Infrastructure team
+
+<table>
+<tr>
+    <td><b>WG-infra-crates.io</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-infra-perf</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-infra-crater2</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-infra-secure</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-infra-host</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-infra-rustbuild</b></td>
+    <td></td>
```

Comment:
Ensuring stability and speed in the compiler build.

I'm not super happy with this one...